org.adempiere.base/src/org/adempiere/base/IDisplayTypeFactory.java

```diff
@@ -29,6 +29,7 @@
 	public Integer getDefaultPrecision(int displayType);
 	public boolean isText(int displayType);
 	public boolean isDate (int displayType);
+	public boolean isList (int displayType);
```

Comment:
This change breaks the plugins that use the interface.

I suggest the following change for better backward compatibility:
[default_interface.txt](https://github.com/idempiere/idempiere/files/5743146/default_interface.txt)